notebooks/Emukit-tutorial-bayesian-optimization-context-variables.ipynb

```diff
@@ -0,0 +1,248 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bayesian optimization with context variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we are going to see how to used Emukit to solve optimization problems in which certain variables are fixed during the optimization phase. These are called context variables. For details see:"
```

Comment:
I think there is a link to a paper missing?
```suggestion
    "In this notebook we are going to see how to use Emukit to solve optimization problems in which certain variables are fixed during the optimization phase. These are called context variables. For details see:"
```

Reply:
added